default.nix

```diff
@@ -0,0 +1,3 @@
+(import (fetchTarball https://github.com/edolstra/flake-compat/archive/master.tar.gz) {
```

Comment:
The version of `flake-compat` should be pinned to what is in `flake.lock`, see the [README of `flake-compat`](https://github.com/edolstra/flake-compat#usage)